en/docs/reference/connectors/google-pubsub-inbound/google-pubsub-inbound-endpoint-example.md

```diff
@@ -0,0 +1,94 @@
+# Google Pub/Sub Inbound Endpoint Example
+
+## Introduction
+Google Cloud Pub/Sub is a globally distributed, asynchronous messaging service designed to decouple services that produce events from services that process events.
+
+The Google Pub/Sub Inbound Endpoint provides a robust and production-grade implementation of a Google Cloud Pub/Sub subscriber using the Streaming Pull API, built on top of Google's high-level Java client library. The Streaming Pull API is ideal for low-latency, high-throughput message consumption, as it maintains a persistent connection to efficiently stream messages from the server to the client.
+
+!!! Note
+    The WSO2 MI Google Pub/Sub Inbound Endpoint also supports subscription creation. To create or update a subscription, the service account must have the **roles/pubsub.editor** role. However, if you only intend to consume messages from an existing subscription, the **roles/pubsub.subscriber** role is sufficient.
+
+## What you’ll build
+
+By following this tutorial, you will gain hands-on experience in:
+
+1. Setting up Google Cloud Pub/Sub resources
+2. Configure the Inbound Endpoint in WSO2 Micro Integrator using the Visual Studio Code extension
+3. Run and test the integration to receive real-time messages
```

Comment:
```suggestion
3. Running and testing the integration to receive real-time messages.
```

Reply:
Shouldn't we use imperative mode for titles?